installer/pkg/components/database/external/init/00-create-and-init-sessions-db.sql

```diff
@@ -0,0 +1,16 @@
+-- Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+-- Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+-- must be idempotent
+
+CREATE DATABASE IF NOT EXISTS `gitpod-sessions` CHARSET utf8mb4;
+
+USE `gitpod-sessions`;
+
+CREATE TABLE IF NOT EXISTS sessions (
```

Comment:
we might want to extra those init files to a `pkg/database/init` package.

Reply:
I've exported the `external` and `cloudsql` init to a `pkg/database/init` package. As the `incluster` init scripts are subtly different, I've left those out but we may want to incorporate that in future